Formula/p/passt.rb

```diff
@@ -0,0 +1,31 @@
+class Passt < Formula
+  desc "User-mode networking daemons for virtual machines and namespaces"
+  homepage "https://passt.top/passt/about/"
+  url "https://passt.top/passt/snapshot/passt-2024_07_26.57a21d2.tar.xz"
+  version "2024_07_26.57a21d2"
```

Comment:
No idea about version number. Repology is all over the place https://repology.org/project/passt/versions

Currently just used default version, i.e. the version embedded when building via git checkout - https://passt.top/passt/tree/Makefile#n12
```make
VERSION ?= $(shell git describe --tags HEAD 2>/dev/null || echo "unknown\ version")
```

Reply:
We use git tags for upstream versions (which are announced on the [passt-user](https://lists.passt.top/postorius/lists/passt-user.passt.top/) mailing list), latest is [2024_08_06.ee36266](https://archives.passt.top/passt-user/20240806162144.18a23a3b@elisabeth/), and I would recommend using that one to start with.

Repology shows very different version strings between distributions because some distributions require additional markers or separators for packages that include a date and git information in the version name (as opposed to just numbers).

See for instance Debian's [watch](https://salsa.debian.org/sbrivio/passt/-/blob/master/debian/watch) file, Chimera's [template](https://github.com/chimera-linux/cports/blob/master/contrib/passt/template.py), Void Linux's [template](https://github.com/void-linux/void-packages/blob/master/srcpkgs/passt/template), Arch's [nvchecker](https://gitlab.archlinux.org/archlinux/packaging/packages/passt/-/blob/main/.nvchecker.toml) file.